Makefile

```diff
@@ -295,10 +297,19 @@ e2e-notation-setup:
 	${GITHUB_WORKSPACE}/bin/oras cp --from-oci-layout .staging/notation/notation.tar:v0 ${TEST_REGISTRY}/notation:unsigned
 	rm .staging/notation/notation.tar
 
+	printf 'FROM ${ALPINE_IMAGE}\nCMD ["echo", "notation tsa signed image"]' > .staging/notation/Dockerfile
+	docker buildx create --use
+	docker buildx build --output type=oci,dest=.staging/notation/notation.tar -t notation:v0 .staging/notation
+	${GITHUB_WORKSPACE}/bin/oras cp --from-oci-layout .staging/notation/notation.tar:v0 ${TEST_REGISTRY}/notation:tsa
+	rm .staging/notation/notation.tar
+	
 	rm -rf ~/.config/notation
 	.staging/notation/notation cert generate-test --default "ratify-bats-test"
+	mkdir -p .staging/tsa
+	curl -L https://github.com/notaryproject/notation/raw/a034721a7e3088250bbb04c5fccedbfca966d49e/internal/testdata/tsaRootCA.cer --output ${GITHUB_WORKSPACE}/.staging/tsa/tsaroot.cer
```

Comment:
wonder if we can just keep a new one in Ratify repo in case notation project modifies the cert?